packages/theming/atlas/scripts/build.js

```diff
@@ -41,7 +48,72 @@ async function main() {
     mkdir("-p", join(outputDir, "themesource/atlas_web_content"));
     mkdir("-p", join(outputDir, "themesource/atlas_nativemobile_content"));
 
-    await buildAndCopyAtlas(mode === "start", outputDir);
+    if (mode === "start") {
+        if (process.argv.includes("--validate-sass")) {
```

Comment:
What about making this (`--validate-sass`) a constant?

Reply:
I extracted the result in a constant.